xblock/exceptions.py

```diff
@@ -30,8 +30,9 @@ def __init__(self, saved_fields, dirty_fields):
         `dirty_fields` - a set of fields that were left dirty after the save
         """
         # Exception is an old-style class, so can't use super
-        Exception.__init__(self)
+        Exception.__init__(self, message)
```

Comment:
Pardon, Now I understand. Either of them is enough. So let go with keeping it in the `__init__`?
What do you think?